components/atlas-uvs/README.md

```diff
@@ -41,6 +42,16 @@ Install and use by directly including the [browser files](dist):
 </body>
 ```
 
+For A-Frame versions 1.2 and later:
+```
+  <script src="https://unpkg.com/aframe-atlas-uvs-component@3.0.0/dist/aframe-atlas-uvs-component.min.js"></script>
+```
+
+For A-Frame versions 1.1 and earlier:
+```
+  <script src="https://unpkg.com/aframe-atlas-uvs-component@2.1.0/dist/aframe-atlas-uvs-component.min.js"></script>
```

Comment:
Does the new version also work with older A-Frame versions? THREE.BufferGeometry is not new on the THREE side.

Reply:
As far as I can tell the default A-Frame plane geometry is non-buffered in 1.1 and earlier, and buffered geo in 1.2 and beyond. The update assumes buffered only (and not handling non-buffer case). In theory it could be possible to support both cases with some sort of "if buffered" logic.

Reply:
simplified readme and only include newer version in examples